pkg/maps/metricsmap/metricsmap.go

```diff
@@ -27,7 +29,11 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+// Metrics is the bpf metrics map
+var Metrics *bpf.Map
 var log = logging.DefaultLogger
+var file *os.File
```

Comment:
Is there a reason for `file` ? It looks like any variable named `file` is created with `:=` in the code.

Reply:
its not needed. I will remove it

Reply:
done